### PR TITLE
L1 PR 10_2 Omtf unpacker and ghost-buster & CaloLayer2 HoEext and Qual bit in ntuple

### PR DESCRIPTION
**Pull request by rekovic**
PR 10_2  Fixes in L1T

Contains: 
 * OMTF unpacker in order assumed by GMT  (v97.23)
 * OMTF emulator ghost-buster second implementation (v97.28)
 * CaloLayer2 HoEext and Qual bit in ntuple (v97.26)

### CODE REVIEW

**Comment by rekovic** [comment]
please test

**Comment by rekovic** [comment]
please test

**Comment by rekovic** [comment]
+1

File: L1Trigger/L1TMuonOverlap/python/simOmtfDigis_cfi.py [hunk 1]
```diff
@@ -14,7 +14,8 @@
                               eventsXMLFiles = cms.vstring("TestEvents.xml"),
                               dropRPCPrimitives = cms.bool(False),                                    
                               dropDTPrimitives = cms.bool(False),                                    
-                              dropCSCPrimitives = cms.bool(False)   
+                              dropCSCPrimitives = cms.bool(False),
+                              #ghostBusterType = cms.string("GhostBusterPreferRefDt")
```
Comment (fabiocos) [inline L1Trigger/L1TMuonOverlap/python/simOmtfDigis_cfi.py]: so is this GhostBuster effectiveley never used so far, and left there for future studies? (How) was it tested?

Comment (fabiocos) [reply L1Trigger/L1TMuonOverlap/python/simOmtfDigis_cfi.py]: @rekovic could you please clarify the point above? Apart for than that I am ok with integrating this PR by this evening

Comment (rekovic) [reply L1Trigger/L1TMuonOverlap/python/simOmtfDigis_cfi.py]: Thanks @fabiocos That is correct. 
The code "as is" was tested with unit tests showing no differences in emulated muons.  "Ghost-busting" when turned on was tested by the developers.

**Comment by fabiocos** [comment]
+1

**Pull request merged**